sdl_android/src/main/java/com/smartdevicelink/api/screen/SoftButtonManager.java

```diff
@@ -546,4 +561,8 @@ private boolean allCurrentStateImagesAreUploaded() {
         }
         return softButtons;
     }
+
+    interface UpdateListener{
```

Comment:
This should be part of the `SoftButtonObject`. The scope of the callback only makes sense in the context of the `SoftButtonObject`, the manager can create an instantiation of the callback (see Android's OnClickListener)

Reply:
Gotcha. I will update that.